src/extras/cacheAsBitmap.js

```diff
@@ -60,12 +68,16 @@ Object.defineProperties(DisplayObject.prototype, {
                 this.getBounds = this._originalGetBounds;
 
                 this.updateTransform = this._originalUpdateTransform;
-                this.containsPoint = this._originalContainesPoint;
+                this.containsPoint = this._originalContainsPoint;
             }
         }
     }
 });
-
+/**
+* Renders a cached version of the sprite with WebGL
+* @param renderer {WebGLRenderer} the WebGL renderer
```

Comment:
Make sure to have a line of space between the description and the jsdoc tags, makes it a lot easier to read.